sbin/submitty_daemon_jobs/submitty_jobs/jobs.py

```diff
@@ -317,6 +318,85 @@ def run_job(self):
         os.chdir(current_path)
 
 
+class UpdateDockerData(AbstractJob):
+    '''
+    Performs the commands 'docker info' & 'docker images' and saves the 
+    results to a json file where the PHP user can read the data, used for
+    the submitty docker interface
+    '''
+    def format_bytes(size):
+        power = 2**10
+        n = 0
+        power_labels = {0 : '', 1: 'KB', 2: 'MB', 3: 'GB', 4: 'TB'}
+        while size > power:
+            size /= power
+            n += 1
+        
+        ret = str(math.floor(size))
+        if n in power_labels:
+            ret += power_labels[n]
+
+        return ret
+
+
+    def run_job(self):
+
+        tgt_path = Path(DATA_DIR, 'docker_data', 'submitty_docker.json')
+        log_path = Path(DATA_DIR, 'logs', 'docker_interface_logs')
+
+        today = datetime.datetime.now()
+        log_file_path = os.path.join(log_path, 
+                                "{:04d}{:02d}{:02d}.txt".format(today.year, today.month,
+                                today.day))
+        full_time = today.strftime("%m/%d/%Y %H:%M:%S")
+
+        docker_client = docker.from_env()
+        try:
+            docker_info = docker_client.info()
+            dokcer_client = docker.APIClient(base_url='unix://var/run/docker.sock')
```

Comment:
`s/dokcer_client/docker_client/`

Also, can you not do `docker.from_env()` and use the higher level client? Or is there something in particular we need from the lower level API client?

Reply:
I was using the lower-level API because the output of collecting images was more similar to the previous `docker images` command that was being run. But yeah like you mentioned in the rest of your review, I can collect the raw data here and format it on the php side.

Reply:
```
docker_client = docker.from_env()
for image in docker_client.images.list():
    print(image.attrs)
```

We do not need the APIClient here, and should be avoided unless we explicitly need it given that you have to pass the `base_url` variable in, which makes it hard to extend, especially if doing stuff like setting docker environment variables and such, which a sysadmin would probably expect to do when dealing with Docker stuff.